nucypher/blockchain/eth/agents.py

```diff
@@ -796,17 +825,16 @@ def post_transcript(
         ritual_id: int,
         transcript: Transcript,
         transacting_power: TransactingPower,
-        fire_and_forget: bool = True,
-    ) -> Union[TxReceipt, HexBytes]:
+    ) -> AsyncTx:
         contract_function: ContractFunction = self.contract.functions.postTranscript(
             ritualId=ritual_id, transcript=bytes(transcript)
         )
-        receipt = self.blockchain.send_transaction(
+        atx = self.blockchain.send_async_transaction(
```

Comment:
Not very important, but I feel that `async_tx` is more descriptive (and also it has more occurences)

Reply:
thanks! ✔️